src/_includes/layouts/main.njk

```diff
@@ -78,6 +78,10 @@ apiDocs: false
           {% endif %}
         >
         <meta name="twitter:image" content="https://user-images.githubusercontent.com/52248161/201438353-17e305a1-c13b-48f6-9bb4-1dde013575a8.jpg" >
+
+        {# Webmentions #}
+        <link rel="pingback" href="https://webmention.io/evently.cloud/xmlrpc" />
+        <link rel="webmention" href="https://webmention.io/evently.cloud/webmention" />
```

Comment:
Need just a few more links here. This allows @mattbishop to claim this site as the owner and lets him log into webmention.io and bridgy.

```html
<link rel="me" href="https://twitter.com/evently_cloud" />
<link rel="me" href="https://twitter.com/MattBishopL3/" />
<link rel="me" href="https://indieweb.social/@Mattbishop/" />
<link rel="me" href="https://github.com/mattbishop/" />
```

After that, we just need to get him to write **1** hello world article to see if it all works ;)

Reply:
Oh I set the `rel="me` on the Twitter and Github `<a>` links in the nav header, is that incorrect or is this just the better more in-depth way?
We could also test with the test articles yeah?

Reply:
You can set the `rel` on any link, so if there's already links in the body that point to any of these you can just add a `rel="me"` there instead!

Reply:
I can work on this today. Do I need to set up an account somewhere? How do I claim the account? Not sure what that means, been unable to follow along this week.

Reply:
You'll need to sign-in with your Evently url at https://webmention.io/, and connect your social media accounts to https://brid.gy/. All of the identifying links have been added to this site via this PR, so you should get the green light when you sign-in to both. If you need any advice past the sign-in point, @evert has expertise about what lies beyond!